requirements.txt

```diff
@@ -1,4 +1,3 @@
-tornado>=3.1.1
```

Comment:
Technically this makes sure our underlying server matches tornado. I couldn't imagine IPython notebook changing (like it did from twisted) again, but still.